c2corg_api/views/document.py

```diff
@@ -433,6 +433,12 @@ def get(self):
         #    # validate permission (authenticated + associated)
         #    # return 403 if not correct
 
+        title = DBSession.query(DocumentLocale.title) \
```

Comment:
This query might fail when you are requesting the history for a language without entry.